modules/aggregation-layers/src/contour-layer/contour-layer.ts

```diff
@@ -351,7 +351,8 @@ export default class ContourLayer<
     const {count} = this.state.weights;
     let {aggregationData} = count;
     if (!aggregationData) {
-      aggregationData = count.aggregationBuffer!.getData() as Float32Array;
+      // @ts-ignore
+      aggregationData = count.aggregationBuffer!.readSyncWebGL2() as Float32Array;
```

Comment:
Flagging in case this bites someone (possibly me) later — return type is Uint8Array, not Float32Array. See https://github.com/visgl/deck.gl/pull/8425.